mermaid-theme.css

```diff
@@ -1,70 +1,146 @@
 .actor, .labelBox {
```

Comment:
@matevz how do you consistently generate `yarn diagrams`? CI seems to always generate a little different. I annoyingly copied them from CI logs diff into here now, but it doesn't print diff inside sapphire-paratime repo

Reply:
This is still an open issue yes. The main issue are font metrics not being available on CI.